tests/foreman/api/test_operatingsystem.py

```diff
@@ -6,11 +6,15 @@
 * http://theforeman.org/api/apidoc/v2/parameters.html
 
 """
-from fauxfactory import gen_integer, gen_utf8
+import random
+from fauxfactory import gen_string
+from httplib import NOT_FOUND
 from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo.constants import OPERATING_SYSTEMS
+from robottelo.datafactory import invalid_values_list, valid_data_list
 from robottelo.decorators import run_only_on, skip_if_bug_open
 from robottelo.test import APITestCase
-from six.moves.http_client import NOT_FOUND
```

Comment:
Six is now a dependency https://github.com/SatelliteQE/robottelo/blob/master/setup.py#L28